_aggregations/metric/geobounds.md

```diff
@@ -26,30 +37,55 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
   }
 }
 ```
+{% include copy-curl.html %}
+
+## Example response
 
-#### Example response
+The response is as follows:
```

Comment:
Could you provide a descriptive sentence for the response?

Reply:
```suggestion
As shown in the following example response, the aggregation returns the `geobounds` containing all `geopoints` in the `geoip.location` field:
```